### ISSUE TITLE
Update MacOS

### ISSUE THREAD

**Issue by constantineoo** [JokerAB/Mechrevo-S1-Hackintosh#2]
Is there an option in config responsible for macos updates. I noticed that Macos 11.2.1 does not appear. Is there any advice for this?
Everything works great !!! big thanks. There is one more little problem. I use a mouse but my touchpad does not work properly, namely the left click key on the touchpad

**Comment by JokerAB** [comment]
Thank you so much for your feedback! I made this EFI in MacOS 11.1. My machine is Mechrevo S1, so I did not test on MSI, 
Sorry for the bad experience.  The left button of my touchpad is normal, but the right button is abnormal.  So my touchpad did not press the buttons, I used gestures instead of buttons. Finally, are you installing MacOS 11.2.1 and don’t see the option?
Looking forward to your feedback again, thank you!

**Comment by constantineoo** [comment]
Yes, when searching for a new version, there is information that the system is up-to-date. When it comes to the touchpad, I only use the mouse anyway.
by entering the update options at the bottom of the Trim profile search. This was not the case before. I think there is some system-responsible option in the openCore configuration

**Comment by JokerAB** [comment]
Go to the App Store and search for MacOS to get the latest version

**Comment by constantineoo** [comment]
do you know how to turn on HiDpi to make the screen bigger?

**Comment by constantineoo** [comment]
Dear friends, you will never be able to update with this config.plist because you have disabled sip in the csr-active position, you need to set: 0000000 only then you will update the system

**Comment by JokerAB** [comment]
Thank you for your feedback, I have fixed this problem and changed it to E703

**Comment by constantineoo** [comment]
:) there is one more little detail. I don't know how to describe it. There are no system sounds such as minimization etc. I used to have an efi from LGS and the sounds were correct but in many efi I found none. The sound is working properly, there are alerts, the music is playing, but there is no system sound

**Comment by constantineoo** [comment]
> Thank you for your feedback, I have fixed this problem and changed it to E703
this is also a bad value. the SIP system in the system report must be turned on otherwise you will not install any update or search for any

**Comment by constantineoo** [comment]
You can update 0.6.8 ? thank you

**Comment by constantineoo** [comment]
你好嗎？ 有空的時候可以看看設置的新內容嗎？  oc 已經更新到 0.7.0 但是我不知道配置選項，你可以看到 config.plist 的變化
